### ISSUE TITLE
Investigate checking ability to send http requests before creating match on a server

### ISSUE THREAD

**Issue by splewis** [splewis/get5-web#12]
There's a common problem of get5_web_avaliable working over rcon, but the server fails creating all http requests via Steamworks to the web api. Not sure what the best way to do this is, especially since the server-availability checking is done synchronously.

**Comment by splewis** [comment]
Note: it's also entirely possible I ditch using http requests in the plugin entirely and just use LogToGame calls with with a separate process on the web server set to listens to messages logged (over the game's udp logging via logaddress_add).

**Comment by Technoblazed** [comment]
Isn't this fixed? @splewis

**Comment by splewis** [comment]
Nope. There's still no check any http requests from the game server actually reach the web server. The ``get5_web_available`` check will make sure SteamWorks is loaded, but doesn't try to send any request. For example if the api url was missing an http or somehow misformatted, there's no check.
